### DISCUSSION TITLE
Version 1.1.1 Does Not Work.

### DISCUSSION THREAD

**Discussion by yodaluca23** [SpotCompiled/SpotilifeC#2] [Announcements]
The latest version (1.1.1) does not work, the reason is because I injected the Spotilife .deb into the latest Spotify IPA, which is not supported by the latest Spotilife. Hopefully Julio will update it, but in the meantime I am looking for a safe decrypted Spotify version 8.7.46, which is supported by the latest Spotilife. Thank you for being patient. I also have plans to make AltStore and Cydia repos for easier updates.

**Comment by ghost** [comment]
Same here.
